src/field.rs

```diff
@@ -51,6 +51,12 @@ pub struct Element<F: Field> {
     phantom: PhantomData<*const F>,
 }
 
+impl<F: Field> num_traits::identities::One for Element<F> {
```

Comment:
Interesting, how did this come up? I'm a bit hesitant about implementing num traits since user would sort of be working with two APIs vs just ours. (Although their is some num stuff in our public API already, but I just prefer to keep it minimal.)

Reply:
Yeah I can get rid of it - I'd initially written a bunch of methods to handle arithmetic on curve points, but shouldn't be necessary.